JavaScript/JavaScript.sublime-syntax

```diff
@@ -934,6 +933,10 @@ contexts:
     - match: '(?=\s*\{)'
       set: function-block
 
+  meta-function-declaration:
```

Comment:
Yeah, because I kept running into https://github.com/SublimeTextIssues/Core/issues/1577. I should probably delete it.